sphinx/quickstart.py

```diff
@@ -1284,6 +1288,8 @@ def generate(d, overwrite=True, silent=False):
         d['extensions'] = '\n' + indent + extensions + ',\n'
     else:
         d['extensions'] = extensions
+    print(d['extensions'])
```

Comment:
Definitely agree with the print, that was an oversight of mine. What is wrong with the ``d['ext_todo'] = text_type(d['ext_todo'])`` line?